web_src/js/components/DiffCommitSelector.vue

```diff
@@ -179,28 +180,29 @@ export default defineComponent({
      * When a commit is clicked with shift this enables the range
      * selection. Second click (with shift) defines the end of the
      * range. This opens the diff of this range
-     * Exception: first commit is the first commit of this PR. Then
-     * the diff from beginning of PR up to the second clicked commit is
-     * opened
```

Comment:
Is it clear about the new logic's behavior?

Reply:
cbe8552a99